.github/workflows/remove-wheels.yml

```
@@ -51,9 +51,11 @@ jobs:
 
       - name: Query package index for packages
         run: |
+          curl https://raw.githubusercontent.com/scientific-python/upload-nightly-action/main/packages-ignore-from-cleanup.txt --output packages-ignore-from-cleanup.txt
           anaconda show "${ANACONDA_USER}" &> >(grep "${ANACONDA_USER}/") | \
               awk '{print $1}' | \
-              sed 's|.*/||g' > package-names.txt
+              sed 's|.*/||g' | \
+              grep -vf packages-ignore-from-cleanup.txt > package-names.txt
```

Small suggestion: Use the full option names of CLI APIs in CI workflows so that tired people late at night don't have to run `man` or `help` to refresh their memories

```suggestion
              grep --invert-match --file packages-ignore-from-cleanup.txt > package-names.txt
```

---

sure. Though `-vf` one is so much a muscle memory, I assume for anyone using shell scripting, that changing it to `--invert-match` I would be the one needing to check the help.

---

Using full option names also keeps thing from falling into the alphabet soup of options (here I'm thinking of something like `tar` where using the UNIX-style option flags means that I'm constantly reaching for `man tar` or `cheat tar`).

---

I think these command line options are so commonly used that it is fine to keep the short version. If the command used more esoteric grep options the long version would be more helpful, but like @bsipocz I would have to think twice if I saw --invert-match, I didn't even know that was the long version of `-v`.